### PR TITLE
[feat]: add separation of resources on ksm and forwarder (recreation of PR #1115) 

### PR DESCRIPTION
**Pull request by csongnr**
## Description

THIS IS A RECREATION OF THIS PR: https://github.com/newrelic/nri-kubernetes/pull/1115 
## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] Add changelog entry following the [contributing guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +43,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Updated github.com/prometheus/common to v0.59.1 - [Changelog 🔗](https://github.com/prometheus/common/releases/tag/v0.59.1)
 - Updated alpine to v3.20.3
 
+
```
Comment (dbudziwojskiNR) [inline CHANGELOG.md:46]: extra new line

**Review by dbudziwojskiNR — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### enhancement 
+- allow separation of resource settings on KSM and forwarder
```
Comment (dbudziwojskiNR) [inline CHANGELOG.md:11]: You need to follow the changelog convention

Comment (csongnr) [reply CHANGELOG.md:11]: i am: https://github.com/newrelic/release-toolkit/blob/main/README.md#validate-markdown 

once its merged, it creates the fancy header w/ the emojis and things

Comment (csongnr) [reply CHANGELOG.md:11]: https://github.com/newrelic/release-toolkit/blob/main/README.md#render-markdown-and-update-markdown 
in the step render-markdown

Comment (dbudziwojskiNR) [reply CHANGELOG.md:11]: I was told it had to be like this: [PR Guide](https://github.com/newrelic/nri-kubernetes/blob/main/CONTRIBUTING.md#pull-requests)

Would need to be like this: 

```
### enhancement 
- Allow separation of resource settings on KSM and forwarder @jddcarreira [#1130](https://github.com/newrelic/nri-kubernetes/pull/1130)
```

Comment (csongnr) [reply CHANGELOG.md:11]: oh i see what you're saying 
ok will update

**Review by dbudziwojskiNR — APPROVED** [review]

**Pull request merged**